apps/src/music/views/hooks/useUpdateAnalytics.tsx

```diff
@@ -0,0 +1,114 @@
+import {useEffect} from 'react';
```

Comment:
Why is this a .tsx file and not a .ts file similar to `useUpdatePlayer.ts`?

Reply:
Good catch :) should be a `.ts` file!